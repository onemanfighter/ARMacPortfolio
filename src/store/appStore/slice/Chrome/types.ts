export type ChromeState = {};

export interface ChromeAppAction {}

export type ChromeStateSlice = ChromeState & ChromeAppAction;
