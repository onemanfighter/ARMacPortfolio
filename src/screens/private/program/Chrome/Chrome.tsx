import { ChromeProps } from './type';

const Chrome = (props: ChromeProps) => {
  return (
    <div>
      <h1>Chrome</h1>
    </div>
  );
};

export default Chrome;
