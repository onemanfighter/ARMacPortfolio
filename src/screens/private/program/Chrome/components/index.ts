export { default as LeftChromeIcons } from './LeftChromeIcons';
