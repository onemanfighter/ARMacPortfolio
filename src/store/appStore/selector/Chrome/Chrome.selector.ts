import { AppStoreState } from '../../appStore';

const chromeSelector = (state: AppStoreState) => ({
  linkStack: state.Chrome.linkStack,
  currentLink: state.Chrome.linkStack[state.Chrome.linkStack.length - 1],
  addLinkStack: state.Chrome.addLinkStack,
  popLinkStack: state.Chrome.removeLinkStack,
  clearLinkStack: state.Chrome.clearLinkStack,
});

export { chromeSelector };
