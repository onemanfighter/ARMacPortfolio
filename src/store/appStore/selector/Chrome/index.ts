export { chromeSelector } from './Chrome.selector';
