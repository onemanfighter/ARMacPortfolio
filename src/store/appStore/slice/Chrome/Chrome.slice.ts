import { AppStoreSlice } from '../../appStore';
import { ChromeState, ChromeStateSlice } from './types';

const defaultChromeState: ChromeState = {
  homepageLink: 'http://www.google.com/webhp?igu=1',
};

const createChromeSlice: AppStoreSlice<ChromeStateSlice> = (set) => ({
  ...defaultChromeState,
});

export default createChromeSlice;
