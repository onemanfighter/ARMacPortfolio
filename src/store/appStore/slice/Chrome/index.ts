export { default as createChromeSlice } from './Chrome.slice';
export type { ChromeStateSlice, ChromeState } from './types';
