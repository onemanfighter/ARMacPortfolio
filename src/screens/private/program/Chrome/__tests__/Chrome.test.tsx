import { fireEvent, render, screen } from '@testing-library/react';
import Chrome from '../Chrome';

describe('Chrome', () => {
  it('should render correctly to match snapshot', () => {
    const { container } = render(<Chrome />);

    expect(container).toMatchSnapshot();
  });

  it('should invoke click handler on back press', () => {
    const { container } = render(<Chrome />);

    expect(container).toMatchSnapshot();
    fireEvent.click(screen.getByLabelText('chrome-back'));
  });

  // it('should invoke click handler ', () => {
  //   const { container } = render(<Chrome />);

  //   fireEvent.change(screen.getByLabelText('chrome-input'), {
  //     target: { value: 'https://www.google.com' },
  //   });

  //   expect(container).toMatchSnapshot();
  // });

  it('should invoke click handler on forward press', () => {
    const { container } = render(<Chrome />);

    fireEvent.click(screen.getByLabelText('chrome-forward'));

    expect(container).toMatchSnapshot();
  });

  it('should invoke click handler on refresh press', () => {
    const { container } = render(<Chrome />);

    fireEvent.click(screen.getByLabelText('chrome-refresh'));

    expect(container).toMatchSnapshot();
  });

  it('should invoke click handler on menu press', () => {
    const { container } = render(<Chrome />);

    fireEvent.click(screen.getByLabelText('chrome-menu'));

    expect(container).toMatchSnapshot();
  });
});
