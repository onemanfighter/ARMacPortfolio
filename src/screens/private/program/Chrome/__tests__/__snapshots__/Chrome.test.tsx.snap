// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Chrome should invoke click handler  1`] = `
<div>
  <div
    aria-label="chrome"
    class="css-10klw3m"
  >
    <div
      class="css-v2fv83"
    >
      <div
        class="css-14ggwnl"
      >
        <button
          aria-label="chrome-back"
          class="chakra-button css-tvciyu"
          type="button"
        >
          <svg
            aria-hidden="true"
            color="#000000"
            fill="currentColor"
            focusable="false"
            height="16px"
            viewBox="0 0 1024 1024"
            width="16px"
          >
            <path
              d="M872 474H286.9l350.2-304c5.6-4.9 2.2-14-5.2-14h-88.5c-3.9 0-7.6 1.4-10.5 3.9L155 487.8a31.96 31.96 0 000 48.3L535.1 866c1.5 1.3 3.3 2 5.2 2h91.5c7.4 0 10.8-9.2 5.2-14L286.9 550H872c4.4 0 8-3.6 8-8v-60c0-4.4-3.6-8-8-8z"
            />
          </svg>
        </button>
        <button
          aria-label="chrome-forward"
          class="chakra-button css-tvciyu"
          type="button"
        >
          <svg
            aria-hidden="true"
            color="#000000"
            fill="currentColor"
            focusable="false"
            height="16px"
            viewBox="0 0 1024 1024"
            width="16px"
          >
            <path
              d="M869 487.8L491.2 159.9c-2.9-2.5-6.6-3.9-10.5-3.9h-88.5c-7.4 0-10.8 9.2-5.2 14l350.2 304H152c-4.4 0-8 3.6-8 8v60c0 4.4 3.6 8 8 8h585.1L386.9 854c-5.6 4.9-2.2 14 5.2 14h91.5c1.9 0 3.8-.7 5.2-2L869 536.2a32.07 32.07 0 000-48.4z"
            />
          </svg>
        </button>
        <button
          aria-label="chrome-refresh"
          class="chakra-button css-tvciyu"
          type="button"
        >
          <svg
            aria-hidden="true"
            color="#000000"
            fill="none"
            focusable="false"
            height="16px"
            stroke="currentColor"
            stroke-linecap="round"
            stroke-linejoin="round"
            stroke-width="2"
            viewBox="0 0 24 24"
            width="16px"
          >
            <path
              d="M0 0h24v24H0z"
              stroke="none"
            />
            <path
              d="M4.05 11a8 8 0 11.5 4m-.5 5v-5h5"
            />
          </svg>
        </button>
      </div>
      <input
        aria-label="chrome-input"
        class="chakra-input css-ei81wc"
        value="https://www.google.com"
      />
      <button
        aria-label="chrome-menu"
        class="chakra-button css-tvciyu"
        type="button"
      >
        <svg
          aria-hidden="true"
          color="#000000"
          fill="currentColor"
          focusable="false"
          height="20px"
          viewBox="0 0 21 21"
          width="20px"
        >
          <g
            fill="currentColor"
            fill-rule="evenodd"
          >
            <path
              d="M11.5 10.5 A1 1 0 0 1 10.5 11.5 A1 1 0 0 1 9.5 10.5 A1 1 0 0 1 11.5 10.5 z"
            />
            <path
              d="M11.5 5.5 A1 1 0 0 1 10.5 6.5 A1 1 0 0 1 9.5 5.5 A1 1 0 0 1 11.5 5.5 z"
            />
            <path
              d="M11.5 15.5 A1 1 0 0 1 10.5 16.5 A1 1 0 0 1 9.5 15.5 A1 1 0 0 1 11.5 15.5 z"
            />
          </g>
        </svg>
      </button>
    </div>
    <iframe
      aria-label="github"
      height="100%"
      src="http://www.google.com/webhp?igu=1"
      width="100%"
    />
  </div>
</div>
`;

exports[`Chrome should invoke click handler on back press 1`] = `
<div>
  <div
    aria-label="chrome"
    class="css-10klw3m"
  >
    <div
      class="css-v2fv83"
    >
      <div
        class="css-14ggwnl"
      >
        <button
          aria-label="chrome-back"
          class="chakra-button css-tvciyu"
          type="button"
        >
          <svg
            aria-hidden="true"
            color="#000000"
            fill="currentColor"
            focusable="false"
            height="16px"
            viewBox="0 0 1024 1024"
            width="16px"
          >
            <path
              d="M872 474H286.9l350.2-304c5.6-4.9 2.2-14-5.2-14h-88.5c-3.9 0-7.6 1.4-10.5 3.9L155 487.8a31.96 31.96 0 000 48.3L535.1 866c1.5 1.3 3.3 2 5.2 2h91.5c7.4 0 10.8-9.2 5.2-14L286.9 550H872c4.4 0 8-3.6 8-8v-60c0-4.4-3.6-8-8-8z"
            />
          </svg>
        </button>
        <button
          aria-label="chrome-forward"
          class="chakra-button css-tvciyu"
          type="button"
        >
          <svg
            aria-hidden="true"
            color="#000000"
            fill="currentColor"
            focusable="false"
            height="16px"
            viewBox="0 0 1024 1024"
            width="16px"
          >
            <path
              d="M869 487.8L491.2 159.9c-2.9-2.5-6.6-3.9-10.5-3.9h-88.5c-7.4 0-10.8 9.2-5.2 14l350.2 304H152c-4.4 0-8 3.6-8 8v60c0 4.4 3.6 8 8 8h585.1L386.9 854c-5.6 4.9-2.2 14 5.2 14h91.5c1.9 0 3.8-.7 5.2-2L869 536.2a32.07 32.07 0 000-48.4z"
            />
          </svg>
        </button>
        <button
          aria-label="chrome-refresh"
          class="chakra-button css-tvciyu"
          type="button"
        >
          <svg
            aria-hidden="true"
            color="#000000"
            fill="none"
            focusable="false"
            height="16px"
            stroke="currentColor"
            stroke-linecap="round"
            stroke-linejoin="round"
            stroke-width="2"
            viewBox="0 0 24 24"
            width="16px"
          >
            <path
              d="M0 0h24v24H0z"
              stroke="none"
            />
            <path
              d="M4.05 11a8 8 0 11.5 4m-.5 5v-5h5"
            />
          </svg>
        </button>
      </div>
      <input
        aria-label="chrome-input"
        class="chakra-input css-ei81wc"
        value="http://www.google.com/webhp?igu=1"
      />
      <button
        aria-label="chrome-menu"
        class="chakra-button css-tvciyu"
        type="button"
      >
        <svg
          aria-hidden="true"
          color="#000000"
          fill="currentColor"
          focusable="false"
          height="20px"
          viewBox="0 0 21 21"
          width="20px"
        >
          <g
            fill="currentColor"
            fill-rule="evenodd"
          >
            <path
              d="M11.5 10.5 A1 1 0 0 1 10.5 11.5 A1 1 0 0 1 9.5 10.5 A1 1 0 0 1 11.5 10.5 z"
            />
            <path
              d="M11.5 5.5 A1 1 0 0 1 10.5 6.5 A1 1 0 0 1 9.5 5.5 A1 1 0 0 1 11.5 5.5 z"
            />
            <path
              d="M11.5 15.5 A1 1 0 0 1 10.5 16.5 A1 1 0 0 1 9.5 15.5 A1 1 0 0 1 11.5 15.5 z"
            />
          </g>
        </svg>
      </button>
    </div>
    <iframe
      aria-label="github"
      height="100%"
      src="http://www.google.com/webhp?igu=1"
      width="100%"
    />
  </div>
</div>
`;

exports[`Chrome should invoke click handler on forward press 1`] = `
<div>
  <div
    aria-label="chrome"
    class="css-10klw3m"
  >
    <div
      class="css-v2fv83"
    >
      <div
        class="css-14ggwnl"
      >
        <button
          aria-label="chrome-back"
          class="chakra-button css-tvciyu"
          type="button"
        >
          <svg
            aria-hidden="true"
            color="#000000"
            fill="currentColor"
            focusable="false"
            height="16px"
            viewBox="0 0 1024 1024"
            width="16px"
          >
            <path
              d="M872 474H286.9l350.2-304c5.6-4.9 2.2-14-5.2-14h-88.5c-3.9 0-7.6 1.4-10.5 3.9L155 487.8a31.96 31.96 0 000 48.3L535.1 866c1.5 1.3 3.3 2 5.2 2h91.5c7.4 0 10.8-9.2 5.2-14L286.9 550H872c4.4 0 8-3.6 8-8v-60c0-4.4-3.6-8-8-8z"
            />
          </svg>
        </button>
        <button
          aria-label="chrome-forward"
          class="chakra-button css-tvciyu"
          type="button"
        >
          <svg
            aria-hidden="true"
            color="#000000"
            fill="currentColor"
            focusable="false"
            height="16px"
            viewBox="0 0 1024 1024"
            width="16px"
          >
            <path
              d="M869 487.8L491.2 159.9c-2.9-2.5-6.6-3.9-10.5-3.9h-88.5c-7.4 0-10.8 9.2-5.2 14l350.2 304H152c-4.4 0-8 3.6-8 8v60c0 4.4 3.6 8 8 8h585.1L386.9 854c-5.6 4.9-2.2 14 5.2 14h91.5c1.9 0 3.8-.7 5.2-2L869 536.2a32.07 32.07 0 000-48.4z"
            />
          </svg>
        </button>
        <button
          aria-label="chrome-refresh"
          class="chakra-button css-tvciyu"
          type="button"
        >
          <svg
            aria-hidden="true"
            color="#000000"
            fill="none"
            focusable="false"
            height="16px"
            stroke="currentColor"
            stroke-linecap="round"
            stroke-linejoin="round"
            stroke-width="2"
            viewBox="0 0 24 24"
            width="16px"
          >
            <path
              d="M0 0h24v24H0z"
              stroke="none"
            />
            <path
              d="M4.05 11a8 8 0 11.5 4m-.5 5v-5h5"
            />
          </svg>
        </button>
      </div>
      <input
        aria-label="chrome-input"
        class="chakra-input css-ei81wc"
        value="http://www.google.com/webhp?igu=1"
      />
      <button
        aria-label="chrome-menu"
        class="chakra-button css-tvciyu"
        type="button"
      >
        <svg
          aria-hidden="true"
          color="#000000"
          fill="currentColor"
          focusable="false"
          height="20px"
          viewBox="0 0 21 21"
          width="20px"
        >
          <g
            fill="currentColor"
            fill-rule="evenodd"
          >
            <path
              d="M11.5 10.5 A1 1 0 0 1 10.5 11.5 A1 1 0 0 1 9.5 10.5 A1 1 0 0 1 11.5 10.5 z"
            />
            <path
              d="M11.5 5.5 A1 1 0 0 1 10.5 6.5 A1 1 0 0 1 9.5 5.5 A1 1 0 0 1 11.5 5.5 z"
            />
            <path
              d="M11.5 15.5 A1 1 0 0 1 10.5 16.5 A1 1 0 0 1 9.5 15.5 A1 1 0 0 1 11.5 15.5 z"
            />
          </g>
        </svg>
      </button>
    </div>
    <iframe
      aria-label="github"
      height="100%"
      src="http://www.google.com/webhp?igu=1"
      width="100%"
    />
  </div>
</div>
`;

exports[`Chrome should invoke click handler on menu press 1`] = `
<div>
  <div
    aria-label="chrome"
    class="css-10klw3m"
  >
    <div
      class="css-v2fv83"
    >
      <div
        class="css-14ggwnl"
      >
        <button
          aria-label="chrome-back"
          class="chakra-button css-tvciyu"
          type="button"
        >
          <svg
            aria-hidden="true"
            color="#000000"
            fill="currentColor"
            focusable="false"
            height="16px"
            viewBox="0 0 1024 1024"
            width="16px"
          >
            <path
              d="M872 474H286.9l350.2-304c5.6-4.9 2.2-14-5.2-14h-88.5c-3.9 0-7.6 1.4-10.5 3.9L155 487.8a31.96 31.96 0 000 48.3L535.1 866c1.5 1.3 3.3 2 5.2 2h91.5c7.4 0 10.8-9.2 5.2-14L286.9 550H872c4.4 0 8-3.6 8-8v-60c0-4.4-3.6-8-8-8z"
            />
          </svg>
        </button>
        <button
          aria-label="chrome-forward"
          class="chakra-button css-tvciyu"
          type="button"
        >
          <svg
            aria-hidden="true"
            color="#000000"
            fill="currentColor"
            focusable="false"
            height="16px"
            viewBox="0 0 1024 1024"
            width="16px"
          >
            <path
              d="M869 487.8L491.2 159.9c-2.9-2.5-6.6-3.9-10.5-3.9h-88.5c-7.4 0-10.8 9.2-5.2 14l350.2 304H152c-4.4 0-8 3.6-8 8v60c0 4.4 3.6 8 8 8h585.1L386.9 854c-5.6 4.9-2.2 14 5.2 14h91.5c1.9 0 3.8-.7 5.2-2L869 536.2a32.07 32.07 0 000-48.4z"
            />
          </svg>
        </button>
        <button
          aria-label="chrome-refresh"
          class="chakra-button css-tvciyu"
          type="button"
        >
          <svg
            aria-hidden="true"
            color="#000000"
            fill="none"
            focusable="false"
            height="16px"
            stroke="currentColor"
            stroke-linecap="round"
            stroke-linejoin="round"
            stroke-width="2"
            viewBox="0 0 24 24"
            width="16px"
          >
            <path
              d="M0 0h24v24H0z"
              stroke="none"
            />
            <path
              d="M4.05 11a8 8 0 11.5 4m-.5 5v-5h5"
            />
          </svg>
        </button>
      </div>
      <input
        aria-label="chrome-input"
        class="chakra-input css-ei81wc"
        value="http://www.google.com/webhp?igu=1"
      />
      <button
        aria-label="chrome-menu"
        class="chakra-button css-tvciyu"
        type="button"
      >
        <svg
          aria-hidden="true"
          color="#000000"
          fill="currentColor"
          focusable="false"
          height="20px"
          viewBox="0 0 21 21"
          width="20px"
        >
          <g
            fill="currentColor"
            fill-rule="evenodd"
          >
            <path
              d="M11.5 10.5 A1 1 0 0 1 10.5 11.5 A1 1 0 0 1 9.5 10.5 A1 1 0 0 1 11.5 10.5 z"
            />
            <path
              d="M11.5 5.5 A1 1 0 0 1 10.5 6.5 A1 1 0 0 1 9.5 5.5 A1 1 0 0 1 11.5 5.5 z"
            />
            <path
              d="M11.5 15.5 A1 1 0 0 1 10.5 16.5 A1 1 0 0 1 9.5 15.5 A1 1 0 0 1 11.5 15.5 z"
            />
          </g>
        </svg>
      </button>
    </div>
    <iframe
      aria-label="github"
      height="100%"
      src="http://www.google.com/webhp?igu=1"
      width="100%"
    />
  </div>
</div>
`;

exports[`Chrome should invoke click handler on refresh press 1`] = `
<div>
  <div
    aria-label="chrome"
    class="css-10klw3m"
  >
    <div
      class="css-v2fv83"
    >
      <div
        class="css-14ggwnl"
      >
        <button
          aria-label="chrome-back"
          class="chakra-button css-tvciyu"
          type="button"
        >
          <svg
            aria-hidden="true"
            color="#000000"
            fill="currentColor"
            focusable="false"
            height="16px"
            viewBox="0 0 1024 1024"
            width="16px"
          >
            <path
              d="M872 474H286.9l350.2-304c5.6-4.9 2.2-14-5.2-14h-88.5c-3.9 0-7.6 1.4-10.5 3.9L155 487.8a31.96 31.96 0 000 48.3L535.1 866c1.5 1.3 3.3 2 5.2 2h91.5c7.4 0 10.8-9.2 5.2-14L286.9 550H872c4.4 0 8-3.6 8-8v-60c0-4.4-3.6-8-8-8z"
            />
          </svg>
        </button>
        <button
          aria-label="chrome-forward"
          class="chakra-button css-tvciyu"
          type="button"
        >
          <svg
            aria-hidden="true"
            color="#000000"
            fill="currentColor"
            focusable="false"
            height="16px"
            viewBox="0 0 1024 1024"
            width="16px"
          >
            <path
              d="M869 487.8L491.2 159.9c-2.9-2.5-6.6-3.9-10.5-3.9h-88.5c-7.4 0-10.8 9.2-5.2 14l350.2 304H152c-4.4 0-8 3.6-8 8v60c0 4.4 3.6 8 8 8h585.1L386.9 854c-5.6 4.9-2.2 14 5.2 14h91.5c1.9 0 3.8-.7 5.2-2L869 536.2a32.07 32.07 0 000-48.4z"
            />
          </svg>
        </button>
        <button
          aria-label="chrome-refresh"
          class="chakra-button css-tvciyu"
          type="button"
        >
          <svg
            aria-hidden="true"
            color="#000000"
            fill="none"
            focusable="false"
            height="16px"
            stroke="currentColor"
            stroke-linecap="round"
            stroke-linejoin="round"
            stroke-width="2"
            viewBox="0 0 24 24"
            width="16px"
          >
            <path
              d="M0 0h24v24H0z"
              stroke="none"
            />
            <path
              d="M4.05 11a8 8 0 11.5 4m-.5 5v-5h5"
            />
          </svg>
        </button>
      </div>
      <input
        aria-label="chrome-input"
        class="chakra-input css-ei81wc"
        value="http://www.google.com/webhp?igu=1"
      />
      <button
        aria-label="chrome-menu"
        class="chakra-button css-tvciyu"
        type="button"
      >
        <svg
          aria-hidden="true"
          color="#000000"
          fill="currentColor"
          focusable="false"
          height="20px"
          viewBox="0 0 21 21"
          width="20px"
        >
          <g
            fill="currentColor"
            fill-rule="evenodd"
          >
            <path
              d="M11.5 10.5 A1 1 0 0 1 10.5 11.5 A1 1 0 0 1 9.5 10.5 A1 1 0 0 1 11.5 10.5 z"
            />
            <path
              d="M11.5 5.5 A1 1 0 0 1 10.5 6.5 A1 1 0 0 1 9.5 5.5 A1 1 0 0 1 11.5 5.5 z"
            />
            <path
              d="M11.5 15.5 A1 1 0 0 1 10.5 16.5 A1 1 0 0 1 9.5 15.5 A1 1 0 0 1 11.5 15.5 z"
            />
          </g>
        </svg>
      </button>
    </div>
    <iframe
      aria-label="github"
      height="100%"
      src="http://www.google.com/webhp?igu=1"
      width="100%"
    />
  </div>
</div>
`;

exports[`Chrome should render correctly to match snapshot 1`] = `
<div>
  <div
    aria-label="chrome"
    class="css-10klw3m"
  >
    <div
      class="css-v2fv83"
    >
      <div
        class="css-14ggwnl"
      >
        <button
          aria-label="chrome-back"
          class="chakra-button css-tvciyu"
          type="button"
        >
          <svg
            aria-hidden="true"
            color="#000000"
            fill="currentColor"
            focusable="false"
            height="16px"
            viewBox="0 0 1024 1024"
            width="16px"
          >
            <path
              d="M872 474H286.9l350.2-304c5.6-4.9 2.2-14-5.2-14h-88.5c-3.9 0-7.6 1.4-10.5 3.9L155 487.8a31.96 31.96 0 000 48.3L535.1 866c1.5 1.3 3.3 2 5.2 2h91.5c7.4 0 10.8-9.2 5.2-14L286.9 550H872c4.4 0 8-3.6 8-8v-60c0-4.4-3.6-8-8-8z"
            />
          </svg>
        </button>
        <button
          aria-label="chrome-forward"
          class="chakra-button css-tvciyu"
          type="button"
        >
          <svg
            aria-hidden="true"
            color="#000000"
            fill="currentColor"
            focusable="false"
            height="16px"
            viewBox="0 0 1024 1024"
            width="16px"
          >
            <path
              d="M869 487.8L491.2 159.9c-2.9-2.5-6.6-3.9-10.5-3.9h-88.5c-7.4 0-10.8 9.2-5.2 14l350.2 304H152c-4.4 0-8 3.6-8 8v60c0 4.4 3.6 8 8 8h585.1L386.9 854c-5.6 4.9-2.2 14 5.2 14h91.5c1.9 0 3.8-.7 5.2-2L869 536.2a32.07 32.07 0 000-48.4z"
            />
          </svg>
        </button>
        <button
          aria-label="chrome-refresh"
          class="chakra-button css-tvciyu"
          type="button"
        >
          <svg
            aria-hidden="true"
            color="#000000"
            fill="none"
            focusable="false"
            height="16px"
            stroke="currentColor"
            stroke-linecap="round"
            stroke-linejoin="round"
            stroke-width="2"
            viewBox="0 0 24 24"
            width="16px"
          >
            <path
              d="M0 0h24v24H0z"
              stroke="none"
            />
            <path
              d="M4.05 11a8 8 0 11.5 4m-.5 5v-5h5"
            />
          </svg>
        </button>
      </div>
      <input
        aria-label="chrome-input"
        class="chakra-input css-ei81wc"
        value="http://www.google.com/webhp?igu=1"
      />
      <button
        aria-label="chrome-menu"
        class="chakra-button css-tvciyu"
        type="button"
      >
        <svg
          aria-hidden="true"
          color="#000000"
          fill="currentColor"
          focusable="false"
          height="20px"
          viewBox="0 0 21 21"
          width="20px"
        >
          <g
            fill="currentColor"
            fill-rule="evenodd"
          >
            <path
              d="M11.5 10.5 A1 1 0 0 1 10.5 11.5 A1 1 0 0 1 9.5 10.5 A1 1 0 0 1 11.5 10.5 z"
            />
            <path
              d="M11.5 5.5 A1 1 0 0 1 10.5 6.5 A1 1 0 0 1 9.5 5.5 A1 1 0 0 1 11.5 5.5 z"
            />
            <path
              d="M11.5 15.5 A1 1 0 0 1 10.5 16.5 A1 1 0 0 1 9.5 15.5 A1 1 0 0 1 11.5 15.5 z"
            />
          </g>
        </svg>
      </button>
    </div>
    <iframe
      aria-label="github"
      height="100%"
      src="http://www.google.com/webhp?igu=1"
      width="100%"
    />
  </div>
</div>
`;
